### PR TITLE
Resolve TS2531: Object is possibly 'null'

### PR DESCRIPTION
**Pull request by rak-phillip**
This resolves the TS2531: Object is possibly 'null' message that started popping up in my environment. 

I suspect that these warnings are related to eslint updates in #1778

### CODE REVIEW

File: src/window/index.ts [hunk 1]
```diff
@@ -116,12 +116,18 @@ export async function openFirstRun() {
       },
     });
 
+  if (!window) {
```
Comment (rak-phillip) [inline src/window/index.ts:119]: I'm thinking that we might want to log an error if we can't get a window for any reason

**Comment by rak-phillip** [comment]
Also seeing this in our builds:

https://github.com/rancher-sandbox/rancher-desktop/runs/5514988120?check_suite_focus=true

**Comment by rak-phillip** [comment]
Closing as this isn't an issue outside of #1789, bad branching is to blame on this one..

**Comment by mook-as** [comment]
As far as I can tell, this only happens on #1789, because TypeScript couldn't tell that the early return from `createWindow` would only happen if `window` is not `null` (because it just sees a call to `didRestoreWindow`).

This shouldn't be needed on `main`, and will come in as part of #1789.

**Pull request closed without merging**